packages/consumption/test/modules/requests/RequestsIntegrationTest.ts

```diff
@@ -89,13 +89,22 @@ export class RequestsTestsContext {
             context.mockEventBus,
             { address: CoreAddress.from("anAddress") },
             {
-                getActiveRelationshipToIdentity: () => Promise.resolve(context.relationshipToReturnFromGetActiveRelationshipToIdentity)
+                getRelationshipToIdentity: () => Promise.resolve(context.relationshipToReturnFromGetRelationshipToIdentity)
```

Comment:
you can only work with requests if there is an active relationship... I don't really get why you renamed / changed all of this.

Reply:
got it now.